src/solver/utils/ortools_utils.h

```diff
@@ -48,6 +49,26 @@ namespace Antares
 namespace Optimization
 {
 
+class OrtoolsLogHandler : public LogHandlerInterface
+{
+public:
+    // tmp test with std::cout
+    explicit OrtoolsLogHandler();
+    void message(const char* msg) override
+    {
+        log_writer_ << msg << std::endl;
+    }
+
+private:
+    // TODO won't work in ci, needs ortools update
+    // see https://github.com/rte-france/or-tools/pull/112
+
+    // // TODO
+    std::ofstream log_writer_;
+    // log_writer.open(log_file_per_thread, std::ofstream::out | std::ofstream::app);
+    // log_streams.push_back(&log_writer);
+};
+void setOrtoolsSolverLogs(MPSolver* solver, OrtoolsLogHandler &ortools_logger);
```

Comment:
```suggestion
void setOrtoolsSolverLogs(MPSolver* solver, LogHandlerInterface &ortools_logger);
```

This function is not really necessary, you may call `EnableOutput` directly